addons/skin.estuary/xml/Variables.xml

```diff
@@ -172,7 +172,7 @@
 		<value condition="ListItem.Property(addon.orphaned)">icons/addonstatus/orphan.png</value>
 		<value condition="ListItem.Property(addon.downloading)">icons/addonstatus/install.png</value>
 		<value condition="ListItem.Property(addon.isinstalled) + !ListItem.Property(addon.isenabled) + Window.IsActive(addonbrowser)">icons/addonstatus/disable.png</value>
-		<value condition="ListItem.Property(addon.hasupdate)">icons/addonstatus/update.png</value>
+		<value condition="ListItem.Property(addon.hasupdate) | ListItem.Property(addon.isupdate)">icons/addonstatus/update.png</value>
```

Comment:
not sure about this...
shouldn't it be `ListItem.Property(addon.hasupdate) + ListItem.Property(addon.isupdate)` ?

assuming the definition of those properties hasn't changed:
`addon.hasupdate`: return **True** when there's an update available for the selected addon
`addon.isupdate`: return **True** if this add-on is a valid update of an installed outdated add-on

i think both should be True if we show the icon?

Reply:
both properties are set here:
https://github.com/xbmc/xbmc/blob/6c2900b9eb3f743b286fe3d5adc9bf6f82968006/xbmc/filesystem/AddonsDirectory.cpp#L826-L827
and now we're looking at addonId, origin and version when checking for outdated addon or update.

assuming we have `Sticky Matrix 1.0.1` installed and look at AddonBrowser (doesn't matter if `My Addons` or `Install from repo`)
`Sticky Matrix 1.0.1` -> hasUpdate = **true**; isUpdate = **false**;
`Sticky Matrix 1.0.2` -> hasUpdate = **false**; _(it is the highest version available)_; isUpdate = **true**;

that's why a logical _OR_ is correct imo.

Reply:
So essentially it means that this addon is an update or it’s an updated addon that has an update?

Reply:
> essentially it means that this addon is an update or it’s an outdated addon that has an update?

exactly like this